### ISSUE TITLE
Breeze: Add nicer error for BREEZE.rst update static check

### ISSUE THREAD

**Issue by potiuk** [apache/airflow#22908]
If you modify BREEZE.rst or any of the "dev" files there is a static check run that updates screenshots in the docs.

If you have no Breeze installed, It should nicely inform the user that they have to install Breeze and redirect them to the documentation on how to do it.

But currently, it will fail with this message:

```
Update output of breeze commands in BREEZE.rst...........................................Failed
- hook id: update-breeze-file
- exit code: 1

Traceback (most recent call last):
  File "./scripts/ci/pre_commit/pre_commit_breeze_cmd_line.py", line 81, in <module>
    print_help_for_all_commands()
  File "./scripts/ci/pre_commit/pre_commit_breeze_cmd_line.py", line 46, in print_help_for_all_commands
    check_call(["breeze", "--help"], env=env)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/subprocess.py", line 358, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/subprocess.py", line 339, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'breeze': 'breeze'
````

This should be fixed.

**Comment by edithturn** [comment]
Thank you, @potiuk I would like to help with this, let me do some research

**Comment by edithturn** [comment]
@potiuk I reproduced the error.
I added breeze installation on static-checks-basic-checks-only, now this step is being skipped, and I don't think it's his purpose to skip it. And static checks are failing with a weird error, looks like everything was fine except after we are adding the images.
https://github.com/apache/airflow/runs/5979380037?check_suite_focus=true

I will give another shoot modifying something on Breeze.rst.

**Comment by potiuk** [comment]
Closed by mistake :)

**Comment by edithturn** [comment]
Got it! I thought it was solved with Bowrna's PR, adding breeze installation in static-checks steps.

**Comment by potiuk** [comment]
Not for the message :)

**Comment by potiuk** [comment]
Actually I found a way to run breeze in pre-commit even if you DO NOT have it installed. Since we are using it in a very limited way, we just need "rich" and "rich-click" addded in the .pre-commit environment, adding a PATH to breeze and we should be able to run breeze to get --help output as well as to check which configuration is available. Fix is coming in  #23570
